pkgs/tools/security/chntpw/default.nix

```
@@ -27,6 +26,6 @@ stdenv.mkDerivation rec {
     description = "An utility to reset the password of any user that has a valid local account on a Windows system";
     maintainers = with lib.maintainers; [ deepfire ];
     license = licenses.gpl2;
-    platforms = with lib.platforms; linux;
+    platforms = with lib.platforms; [ linux darwin "aarch64-darwin" ];
```

This would generated nested lists.

the following would be right
```suggestion
    platforms = with lib.platforms; linux ++ darwin ++ [ "aarch64-darwin" ];
```

but darwin is duplicated

```suggestion
    platforms = with lib.platforms; linux ++ darwin;
```

and linux + darwin is unix

```suggestion
    platforms = lib.platforms.unix;
```

---

Thanks for this! This makes a lot more sense to me now :).